### ISSUE TITLE
Sound stops when audio source is changed

### ISSUE THREAD

**Issue by CreeperPookie** [HielkeMinecraft/OpenNoteBlockStudio#244]
**Describe the bug**
I use this program all the time. However, whenever my wireless headphones disconnect and reconnect, the sounds stop and the app has to be restarted for any sound.

**To Reproduce**
1. Start OpenNoteBlockStudio
2. Disconnect headphones
3. Reconnect headphones
4. Play a note

**Expected behavior**
I think this program should be able to detect when an audio source has changed and use it, instead of having to restart the program every time.

**Additional context**
Note: I know this is similar to #243, but I think this is a different case as the sound stops instead of plays from the wrong device.

**Comment by MericGit** [comment]
Do your headphones have any custom sound cards or drivers by any chance? The only time NBS stops playing sound back is whenever I swap my audio interface so I'm wondering if that's the case

**Comment by CreeperPookie** [comment]
> Do your headphones have any custom sound cards or drivers by any chance? The only time NBS stops playing sound back is whenever I swap my audio interface so I'm wondering if that's the case

Nope, just normal Bluetooth headphones (_Sony_ `WH-CH510` to be exact).

**Comment by chenxi050402** [comment]
Minecraft Java Edition also has this problem.

**Comment by CreeperPookie** [comment]
> Minecraft Java Edition also has this problem.

Yeah, but in Java Edition you can press `F3 + T` to reload textures which also sometimes can bring the sound back.

**Comment by MrSlimeDiamond** [comment]
Same problem here.

**Comment by AdhamHakki** [comment]
It happens with the headphone jack and when my Bluetooth earphones disconnect and reconnect. Please fix this issue. Thank you

**Comment by OctoFlareDev** [comment]
I see this as the same issue as #243 , NBS initializes on the audio device at startup, when the device disappears and reappears, NBS can't initialize again.

**Comment by CreeperPookie** [comment]
> I see this as the same issue as [#243](https://github.com/OpenNBS/NoteBlockStudio/issues/243) , NBS initializes on the audio device at startup, when the device disappears and reappears, NBS can't initialize again.

@OctoFlareDev yeah that's fair, but it is sort of distinct in its own way - for me, it's not always playing on a different device, even when I *reconnect* the original audio source, it doesn't reinitialize for that either
now to be fair, if there was to be a fix for this, it would most likely resolve both issues; but I do think this has a slightly different context too
